engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/views/environment-docker.html

```diff
@@ -0,0 +1,31 @@
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <div data-block="header"></div>
+  <div data-block="selection-summary"></div>
+
+  <div class="well" data-block="filters">
```

Comment:
Still not a fan of the `.well` but if we're going to use it for all filter blocks shouldn't we just put it in bastion?